packages/types/src/index.ts

```diff
@@ -251,6 +251,7 @@ export type Place = {
   name: string;
   networks?: string[];
   rentalVehicle?: { network: string };
+  vehicleRentalStation?: { rentalNetwork: { networkId: string } };
```

Comment:
@daniel-heppner-ibigroup could you take a look at this? Doesn't this imply that the `rentalNetwork` is always there? Is there some way to read this from the graphql?

Reply:
I could put a ? by rentalNetwork. I mean the vehicleRentalStation is optional as it is and each body of data for a rentalStation is going to need to have some reference to what network it's a part of...

![image](https://github.com/user-attachments/assets/1db2013c-28a1-400a-b1de-6c7b1d5200c1)


This applies to when we are planning rental trips that originate or have a leg in the trip that leverages the use of a rental hub. Free floating rental bikes have never been a problem.

Reply:
Just let me know and I can whip those adjustments up in a jiffy! 🥜 🧈

Reply:
Rental network and network ID are required fields in the schema so this is fine!

Reply:
![image](https://github.com/user-attachments/assets/389ea439-ae81-484a-8cd6-746263806779)